packages/applications-service-api/src/validators/request/submission.js

```diff
@@ -0,0 +1,34 @@
+const config = require('../../lib/config');
+const ApiError = require('../../error/apiError');
+const { createRequestValidator } = require('../../utils/openapi');
+
+const validator = createRequestValidator('POST', '/api/v1/submissions/{caseReference}');
+
+const validateRequest = (req) => {
+	const errors = validator.validateRequest(req);
+
+	if (errors) {
+		throw ApiError.badRequest(errors.errors.map((e) => e.message));
+	}
+
+	if (!req.body.representation && !req.file) {
```

Comment:
Do we want to check for on or the other here?

Reply:
i've added a separate check which throws if both `representation` and `file` are provided. That seem ok?

Reply:
https://github.com/Planning-Inspectorate/applications-service/pull/477/files#diff-2b7283eb770ae38117c29db496934d088d842dd6c63690c2bd9e15d9bc868b50R18

Reply:
Yes looks good.